docs/concepts/cluster_queue.md

```diff
@@ -1,8 +1,9 @@
 # Cluster Queue
 
-A `ClusterQueue` is a cluster-scoped object that governs a pool of resources
+A ClusterQueue is a cluster-scoped object that governs a pool of resources
 such as CPU, memory and hardware accelerators. A `ClusterQueue` defines:
-- The resource _flavors_ that it manages, with usage limits and order of consumption.
+- The [resource _flavors_](#resourceflavor-object) that it manages, with usage
```

Comment:
```suggestion
- The [resource _flavors_](#resourceflavor-object) that the cluster manages, with usage
```

I'm assuming "it" refers to "the cluster" but please edit accordingly, but we must mention what "it" means.